doc/ref_cert/lfn/chapters/chapter03.md

```diff
@@ -474,20 +475,23 @@ integrated in
 [Functest Smoke CNTT](https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml)
 
 According to
-[RA1 Core OpenStack Services APIs]({{ "/doc/ref_arch/openstack/chapters/chapter05.html" | relative_url }})
+[RA1 Core OpenStack Services APIs](../../../ref_arch/openstack/chapters/chapter05.md)
 the following test names must not be executed:
 
-| test rejection regular expressions                         | reasons                                            |
-|------------------------------------------------------------|----------------------------------------------------|
-| .\*functional.test_lbaasv2                                 | lbaasv2                                            |
-| .\*RemoteStackTest.test_stack_create_with_cloud_credential | https://gerrit.opnfv.org/gerrit/c/functest/+/69926 |
-| .\*scenario.test_aodh_alarm                                | aodh                                               |
-| .\*tests.scenario.test_autoscaling_lb                      | lbaas                                              |
-| .\*scenario.test_autoscaling_lbv2                          | lbaasv2                                            |
-| .\*scenario.test_server_software_config                    | https://gerrit.opnfv.org/gerrit/c/functest/+/69926 |
-| .\*test_volumes.VolumeBackupRestoreIntegrationTest         | https://gerrit.opnfv.org/gerrit/c/functest/+/69931 |
-| .\*scenario.test_octavia_lbaas                             | octavia                                            |
-| .\*scenario.test_server_cfn_init                           | https://gerrit.opnfv.org/gerrit/c/functest/+/70004 |
+| test rejection regular expressions                                | reasons                                            |
+|-------------------------------------------------------------------|----------------------------------------------------|
```

Comment:
A minor point, but you only need three dashes to create the table structure, i.e. `|---|---|` would result in the same table.

Reply:
I agree, but this change is about changing the links and also there are some of us who would like to keep the tables "aligned" on some level in their source format.

Reply:
no problem - looked like a bunch of different files had been changed, including this line - but you've sorted that now. Resolved.